src/System.Net.Requests/tests/HttpRequestResponseSerializationTest.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization.Formatters.Tests;
+using Xunit;
+
+namespace System.Net.HttpRequest.Tests
+{
+    public partial class HttpRequestResponseSerialization
+    {
+        [OuterLoop]
+        [Theory]
```

Comment:
Right, @justinvp's point is that this test isn't getting executed at all because there is no [MemberData], [InlineData], or anything else providing data to the test... it executes once for each input, and there are no inputs, so it executes zero times.